core/controlplane/config/templates/cloud-config-etcd

```diff
@@ -251,13 +249,12 @@ coreos:
       command: start
       content: |
         [Unit]
-        Wants={{.Etcd.SystemdUnitName}}
+        Requires={{.Etcd.SystemdUnitName}}
```

Comment:
it seems that email responses don't show up here :(

It is `oneshot` service therefore it is never restarted, yet it all worked fine